encoding.opam

```diff
@@ -38,5 +38,5 @@ build: [
 dev-repo: "git+https://github.com/formalsec/encoding.git"
 available: (arch = "x86_64" | arch = "arm64") & os != "win32" & arch != "x86_32"
 pin-depends: [
-  [ "colibri2.5.1.0" "https://git.frama-c.com/pub/colibrics.git#5.1.0"]
+  [ "colibri2.dev" "https://git.frama-c.com/pub/colibrics.git#5.1.0"]
```

Comment:
I think it should be: `"git+https://git.frama-c.com/pub/colibrics.git#5.1.0"` I think without `git+` why he can't fetch the pin

Reply:
True, didn't notice it

Reply:
Well the version is also not right on the colibri2 repo

Reply:
Euh I didn't change it in the template -_-

Reply:
I think we're also missing "colibrilib". The CI is complaning. 

I think with `[ "colibrilib.dev" "git+https://git.frama-c.com/pub/colibrics.git#5.1.0"]` should be enough?